src/main/java/org/spongepowered/api/service/permission/PermissionDescription.java

```diff
@@ -211,8 +259,12 @@
         /**
          * Assigns this permission to the given role-template {@link Subject}.
          *
-         * <p>If the given subject does not exist it will be created. Permission
-         * templates should not be assigned to regular subjects.</p>
+         * <p>Role templates will be namespaced by the plugin that owns each registered permission.
+         * The expected format of the namespaced subject identifier is
+         * {@code  &lt;plugin id>:&lt;role>}. Implementations must provide an un-namespaced role template
```

Comment:
```suggestion
         * {@code <plugin id>:<role>}. Implementations must provide an un-namespaced role template
```
`@code` will be escaped automatically (at least for html)